openpype/lib/log.py

```diff
@@ -102,6 +102,11 @@ def emit(self, record):
 
         except OSError:
             self.handleError(record)
+        except ValueError:
+            # this is raised when logging during interpreter shutdown
+            # or it real edge cases where logging stream is already closed.
+            # In particular, it happens a lot in 3d equalizer.
+            pass
```

Comment:
Do you have an example error stack trace for this? `ValueError` seems like an odd case if the logging stream is closed. I feel like instead there might be issues with the call to `self.format()` or whatever.

Reply:
Sounds like we'd be better of investigating more indeed and make this comment a `# TODO` - I'm a bit worried just capturing `ValueError` here silently will instead be hiding other errors in the future that we do want to know about.